### ISSUE TITLE
make a texture pack that will replicate legacy console in other menus and areas.

### ISSUE THREAD

**Issue by DexrnZacAttack** [Legacy-Edition-Minigames/Minigames#48]
Currently i am working on something like this but i believe yall could do it better than i could, I think yall should make a texture pack that replicates the legacy console menus with the correct sounds, logo, panorama background (with the weird film like thingy over it), and other things.

**Comment by DBTDerpbox** [comment]
menus cant really be changed with a resource pack, sounds can partially be changed

this is planned through the form of a fabric mod, but very low priority at the moment

**Comment by Ucda1212** [comment]
Whilst waiting, [Legacy4j](https://modrinth.com/mod/legacy4j) does exactly what you talk about, and they even have a modpack called [re-console](https://modrinth.com/modpack/legacy-minecraft) which makes it better in every way.

**Comment by S-N00B-1** [comment]
> Whilst waiting, [Legacy4j](https://modrinth.com/mod/legacy4j) does exactly what you talk about, and they even have a modpack called [re-console](https://modrinth.com/modpack/legacy-minecraft) which makes it better in every way.

Yes, we know, it is also currently incompatible with LEM’s UI system. There are talks underway with the Legacy4j devs to resolve this. But it should be mentioned.

**Comment by Ucda1212** [comment]
I did not know this. Sorry for any misunderstanding!
